apps/cookbook/src/app/showcase/data-table-showcase/data-table-showcase.component.html

```diff
@@ -28,10 +12,39 @@ <h2>Data table with minimal data</h2>
   </cookbook-example-viewer>
 
   <h2>Data table wrapped with Kirby Card</h2>
-  <p>Data table wrapped with card and with selectable rows.</p>
+  <p>Data table wrapped with card, with selectable rows.</p>
   <cookbook-example-viewer [html]="dataTableCardExample.template">
     <cookbook-data-table-card-example #dataTableCardExample></cookbook-data-table-card-example>
   </cookbook-example-viewer>
+
+  <h2>Sortable table</h2>
+  <p>
+    Data table wrapped with card, and sorting on header press. We recommend to visit the
```

Comment:
Consider wording and grammar:
```suggestion
    Data table wrapped in a card with sorting. We recommend to visit the
```